### PR TITLE
Update build steps in Makefile

### PR DESCRIPTION
**Pull request by roman3017**
- add steps to build all fs files by default
 - add clean
 - add prog to program blinky

This change enables to run blinky test on TEC0117 with:
```sh
make clean
make
make prog
```

### CODE REVIEW

File: examples/Makefile [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+DEVICE ?= GW1N-9
+PN ?= GW1NR-LV9QN881C6/I5
+
+all: attosoc.fs blinky.fs
+
+%.fs: %.pnr.json
+	gowin_pack -d ${DEVICE} -o $@ $^
+
+%.pnr.json: %.json
+	nextpnr-gowin --json $^ --write $@ --device ${PN} --cst tec0117.cst
```
Comment (pepijndevos) [inline examples/Makefile:10]: The `tec0117.cst` still hardcodes it to that particular board

**Comment by pepijndevos** [comment]
I like the idea of this change, and it's something I was planning to anyway.

I was thinking if you encode the device in the bitstream name, so you can just `make blinky-GW1N-4.fs` and it'd do the right thing. I think you could just have a rule thats `%-GW1N-4.fs` that passes the right part number and cst file?

File: examples/Makefile [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
 	yosys -p "synth_gowin -json $@" $^
 
 %.json: %.v
 	yosys -p "synth_gowin -json $@" $^
+
+prog:
+	openFPGALoader -b littleBee -r -f blinky.fs
+
+clean:
+	rm -f *.json *.fs
```
Comment (pepijndevos) [inline examples/Makefile:22]: I think it's good style to make these targets phony?

File: examples/Makefile (hunk 1 above)
Comment (roman3017) [reply examples/Makefile:10]: Ok, I will make the change.

File: examples/Makefile (hunk 2 above)
Comment (roman3017) [reply examples/Makefile:22]: Ok, I will make the change.

**Pull request merged**